go.mod

```diff
@@ -83,3 +85,5 @@ require (
 	sigs.k8s.io/structured-merge-diff/v4 v4.4.1 // indirect
 	sigs.k8s.io/yaml v1.4.0 // indirect
 )
+
+replace github.com/crossplane/function-sdk-go => github.com/tenitski/function-sdk-go v0.0.0-20250723012926-634ed8aecb6f
```

Comment:
Temporary depending on https://github.com/crossplane/function-sdk-go/pull/218 while that PR is not merged and a new version is not released. Blocks merging of this PR, to be cleaned up and a proper version used instead.